ecms_base/config/install/user.role.content_editor.yml

```diff
@@ -0,0 +1,13 @@
+langcode: en
+status: true
+dependencies: {  }
+id: content_editor
+label: 'Content Editor'
```

Comment:
Should this be named `Content Author` instead to match the roles & workflows documentation?